manage.py

```diff
@@ -6,5 +6,30 @@
     os.environ.setdefault("DJANGO_SETTINGS_MODULE", "config.settings")
 
     from django.core.management import execute_from_command_line
+    from django.conf import settings
+
+    if 'test' in sys.argv:
```

Comment:
Should probably put all this logic inside `config.test_settings` or something, then switch the `DJANGO_SETTINGS_MODULE` when in test.